ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh

```diff
@@ -31,22 +30,21 @@ ssh_private_key_file: ~/.ssh/id_rsa
 ssh_public_key_file: ~/.ssh/id_rsa.pub
 pull_secret: "{{ lookup('file', '../pull_secret.txt') }}"
 bastion_cluster_config_dir: /root/{{ cluster_type }}
-smcipmitool_url:
-bastion_lab_interface: $BASTION_LAB_INTERFACE
 bastion_controlplane_interface: $BASTION_CP_INTERFACE
-controlplane_network: 192.168.216.1/21
-controlplane_network_prefix: 21
-bastion_vlaned_interface: $BASTION_VLANED_INTERFACE
+bastion_lab_interface: $LAB_INTERFACE
+controlplane_lab_interface: $LAB_INTERFACE
 setup_bastion_gogs: false
 setup_bastion_registry: false
 use_bastion_registry: false
-controlplane_lab_interface: $CONTROLPLANE_LAB_INTERFACE
-controlplane_pub_network_cidr:
-controlplane_pub_network_gateway:
 jumbo_mtu: $ENABLE_JUMBO_MTU
 install_rh_crucible: $CRUCIBLE
 rh_crucible_url: "$CRUCIBLE_URL"
 EOF
+
+if [[ $PUBLIC_VLAN == "false" ]]; then
+  echo -e "controlplane_network: 192.168.216.1/21\ncontrolplane_network_prefix: 21" >> /tmp/all.yml
```

Comment:
Its the 2nd time that we have `192.168.216.1` and `21` appearing in the file, should we make vars?
Suggestion:
```
    - name: PRIVATE_SUBNET_ADDRESS
      default: "192.168.216.0"
      documentation: |-
        Private subnet address for OCP nodes. Only considered when PUBLIC_VLAN is set to false
    - name: PRIVATE_SUBNET_MASK
      default: "21"
      documentation: |-
        Private subnet mask for OCP nodes. Only considered when PUBLIC_VLAN is set to false
```

Reply:
They're only present here, no need to introduce new variables